Sources/Core/JavaModelRenderer.swift

```diff
@@ -128,7 +141,7 @@ public struct JavaModelRenderer: JavaFileRenderer {
                 "this." + Languages.java.snakeCaseToPropertyName(param) + " = value;",
             ] }
         }
-        return setters + [JavaIR.method([], "set_bits(int bits)") { [
+        return setters + [JavaIR.method([], "void set_bits(boolean[] bits)") { [
```

Comment:
I had fixed this in a local branch, and didn't realized I hadn't pushed it any PRs. I can split out into a separate PR if we prefer.